lib/honeybadger.ex

```diff
@@ -203,6 +203,7 @@ defmodule Honeybadger do
   """
   @spec notify(Notice.noticeable(), map(), list()) :: :ok
   def notify(exception, metadata \\ %{}, stacktrace \\ []) do
+    {exception, _} = Exception.blame(:error, exception, stacktrace)
```

Comment:
The first argument to `blame/3` is "kind", which isn't always `:error`. However, at this level we don't know whether it is `:error`, `:throw`, or  `{:EXIT, pid}`, so hard coding `:error` works for me.